### ISSUE TITLE
C-x b stops working on X windows after changing Ctrl with xmodmap

### ISSUE THREAD

**Issue by ralt** [ch11ng/exwm#211]
A simple example:

```
$ cat > xmodmaprc <<EOF
remove mod4 = Super_L
add control = Super_L
remove control = Control_L
add mod4 = Control_L
EOF
$ xmodmap xmodmaprc
```

This inverts the control and super keys (very useful on a chromebook). This works fine in emacs buffers, but fails hard on X11 windows: things like `C-t` to open a new tab works in chrome (using the xmodmap config), but `C-x b` to switch buffer doesn't work. All the emacs-related shortcuts stop working.

**Comment by ch11ng** [comment]
The method for detecting modifier keys was not quite right.  I've fixed the problem in ch11ng/xelb@1fafa29ee457ec553e14809c4bc5d4fa59244398.

**Comment by ralt** [comment]
Yay! I'll test that later, thanks for the fix!
